### PR TITLE
fix(quickstart): create app.ts before running tsc --watch

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: public/docs/js/latest/quickstart.jade [hunk 1]
```diff
@@ -26,6 +26,14 @@
     <a href="https://github.com/angular/ts-quickstart"> <code>quickstart</code> GitHub repository</a>.
     This repository provides a faster start than building from <code>npm</code>.
     This repository includes the Angular distribution and type definitions for TypeScript.
+    
+  p.
+    Create two files, <code>index.html</code> and
+    <code>app.ts</code>, both at the root of the project:
+
+  pre.prettyprint
+      $ touch index.html
```
Comment (rkirov) [inline public/docs/js/latest/quickstart.jade]: how about `touch index.html app.ts`

**Comment by rkirov** [comment]
LGTM.

File: public/docs/js/latest/quickstart.jade (hunk 1 above)
Comment (alexeagle) [inline public/docs/js/latest/quickstart.jade]: done

**Comment by alexeagle** [comment]
merged as 1f3940977cf58eb9374a0043f2e34f56ee234740

**Pull request closed without merging**